pke/kyber/internal/common/generic.go

```diff
@@ -39,6 +42,17 @@ func (p *Poly) InvNTT() {
 // Montgomery form.  Products between coefficients of a and b must be strictly
 // bounded in absolute value by 2¹⁵q.  p will be in Montgomery form and
 // bounded in absolute value by 2q.
+//
+// Requires a and b to be in "tangled" order, see Tangle().  p will be in
+// tangled order as well.
 func (p *Poly) MulHat(a, b *Poly) {
 	p.mulHatGeneric(a, b)
 }
+
+// Puts p into the right form to be used with (among others) InvNTT().
+func (p *Poly) Tangle() {
+}
```

Comment:
empty function?

Reply:
In generic code the tangled order is the standard one.  I'll add a comment.